src/components/carbon-provider/carbon-provider.stories.mdx

```diff
@@ -0,0 +1,68 @@
+import { Meta, Story, Preview, Props } from "@storybook/addon-docs/blocks";
+
+import aegeanTheme from "../../style/themes/aegean";
+import baseTheme from "../../style/themes/base";
+import Button from "../button";
+import CarbonProvider from ".";
+
+<Meta title="Carbon Provider" parameters={{ info: { disable: true } }} />
+
+# Carbon Provider
+
+A Carbon Provider uses the `ThemeProvider` from the [styled-components library](https://styled-components.com/docs/advanced#theming) to supply theme styles to your components. Themes are define within the Carbon library.
+
+## Contents
+
+- [Quick Start](#quick-start)
+- [Examples](#examples)
+- [Props](#props)
+
+## Quickstart
+
+```javascript
+import CarbonProvider from "carbon-react/lib/components/carbon-provider";
+```
+
+## Examples
+
+### Default
+
```

Comment:
I think we need some text about mint being the default theme in here

Reply:
Good point, thanks, added